matchms/Spectrum.py

```diff
@@ -234,11 +231,42 @@ def metadata(self, value):
 
     @property
     def losses(self) -> Optional[Fragments]:
-        return self._losses.clone() if self._losses is not None else None
+        return self.compute_losses()
 
-    @losses.setter
-    def losses(self, value: Fragments):
-        self._losses = value
+    def compute_losses(
```

Comment:
Maybe we should remove the option to only calculate the losses between loss_mz_from and loss_mz_to. This made sense before, since it would save memory. However, now I think the masking should just happen in any function that actually works with the losses and this should always return all losses. What do you think?

Reply:
This was not only to save memory, but also too look for losses in a particular range (say 100 to 200 Da).

But I agree, this could easily be done in a filter step after computing the losses.

Reply:
I would probably keep it for now, because I have no good overview of who is using losses for what...

Reply:
Fine with me